recipes/cuda-cupti/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Install to conda style directories
+[[ -d lib64 ]] && mv lib64 lib
+
+[[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
+[[ ${target_platform} == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
+[[ ${target_platform} == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
+
+for i in `ls`; do
+	[[ $i == "build_env_setup.sh" ]] && continue
+	[[ $i == "conda_build.sh" ]] && continue
+	[[ $i == "metadata_conda_debug.yaml" ]] && continue
+	if [[ $i == "lib" ]] || [[ $i == "include" ]]; then
+		mkdir -p ${PREFIX}/${targetsDir}
+		mkdir -p ${PREFIX}/$i
+		cp -rv $i ${PREFIX}/${targetsDir}
+		for j in `ls $i`; do
+			ln -s ../${targetsDir}/$i/$j ${PREFIX}/$i/$j
+		done
+	else
+		cp -rv $i ${PREFIX}
```

Comment:
```suggestion
		cp -rv $i ${PREFIX}
                # Put all other files in targetsDir
               mkdi r -p ${PREFIX}/${targetsDir}/${PKG_NAME}
              cp -rv $i ${PREFIX}/${targetsDir}/${PKG_NAME}		
```
Match libcublas package, and fix the issue where we might create extra directories by mistake